### PR TITLE
fix admin controller

### PR DESCRIPTION
**Pull request by ito1101kai**
1. add a function registered to check if user is already a participant
2. change redirect route after login or registered
3. small fix 2 blade files, privacy and terms.

### CODE REVIEW

File: resources/views/privacy.blade.php [hunk 1]
```diff
@@ -7,11 +7,11 @@
 @endsection
 
 @section('banner')
-<div id="white-wrap" class="py-3">
+<div id="white-wrap" class="pb-3" style="padding-top: calc(72px + 1rem)">
```
Comment (krs91tk) [inline resources/views/privacy.blade.php:10]: if you can its better to move this code to CSS file
`style="padding-top: calc(72px + 1rem)"`

File: resources/views/privacy.blade.php [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
 
     <div class="container mt-3 p-0 text-center">
         <a href="{{ asset('image/text/privacy_policy.pdf') }}" download="{{ asset('image/text/privacy_policy.pdf') }}"
-            style="position: absolute;top: 50px;right: 10%;">Download</a>
+            style="position: absolute;top: calc(72px + 50px);right: 10%;">Download</a>
```
Comment (krs91tk) [inline resources/views/privacy.blade.php:14]: this one too

File: resources/views/terms.blade.php [hunk 3]
```diff
@@ -7,7 +7,7 @@
 @endsection
 
 @section('banner')
-    <div id="white-wrap" class="py-3">
+    <div id="white-wrap" class="class="pb-3" style="padding-top: calc(72px + 1rem)"">
```
Comment (krs91tk) [inline resources/views/terms.blade.php:10]: same here

File: resources/views/privacy.blade.php (hunk 1 above)
Comment (ito1101kai) [reply resources/views/privacy.blade.php:10]: I changed

File: resources/views/privacy.blade.php (hunk 2 above)
Comment (ito1101kai) [reply resources/views/privacy.blade.php:14]: I changed

File: resources/views/terms.blade.php (hunk 3 above)
Comment (ito1101kai) [reply resources/views/terms.blade.php:10]: I changed

**Pull request merged**